File: Android/sukritoberoi_SukritKaurOberoi_2125cse1065_2/GettingStarted/ReverseANumber.java
import java.util.*;
   
   public class Main{
   
   public static void main(String[] args) {
     // write your code here  
     long n;
     Scanner sc=new Scanner(System.in);
     n=sc.nextLong();
     while(n!=0)
     {
         System.out.println(n%10);
         n=n/10;
     }
    }
   }

File: Android/khushi1502_KhushiSachdev_2125cse1077_2/GettingStarted/ReverseANumber.cpp
#include <iostream>

using namespace std;

int main(int argc, char **argv){
    int n;
    cin >> n;
//Write your code here

    while (n != 0){
        int digit = n % 10;
        n = n / 10;
        cout << digit << endl;
    }
}
